### DISCUSSION TITLE
Gathering interest for the Turkish translation of the glossary 🇹🇷

### DISCUSSION THREAD

**Discussion by aliok** [cncf/glossary#2108] [General]
I would like to see if there's enough interest to do the Turkish translation of the glossary.

(I didn't want to go ahead and create a "Initiate a New Localization Team" issue, but I want to have something in this repository to share on social media and gather interest.)

Anybody interested, please say hello in [#glossary-localizations-turkish](https://cloud-native.slack.com/archives/C05ATBGLHJ7) channel in CNCF Slack. You can get an [invitation](https://communityinviter.com/apps/cloud-native/cncf) to join CNCF Slack (open to everyone).

What's required is written in [LOCALIZATION.md](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md), but here's a summary:
- Technical setup (addition of the new language in the site codebase, creating the directory structure, etc) - I can do this part
- Translating the pages in https://glossary.cncf.io/, where each page is a term
  - Minimum: a few mandatory pages (Home, etc) and 10 translated terms (we can't simply translate these minimums and disband though)
- At least three volunteers

To have a feeling for the expected work, feel free to have a look at issues for another language, such as German: https://github.com/cncf/glossary/issues?q=is%3Aissue+label%3Alang%2Fde+

What are the benefits?
- Giving back to the community (both CNCF community and Turkish-speaking software community in general)
- If you are new to open source, this is an opportunity to take a "less complex" first step to make some open source contributions where you learn about open source processes and culture
- If you're a non-tech person, this can be a good way to make some open source contributions, which is always rewarding

See a success story here: https://www.cncf.io/blog/2023/05/18/how-to-become-a-non-tech-non-code-and-non-native-english-speaking-contributor-my-personal-experience/

**Comment by halil-bugol** [comment]
I can support. How do we start? 🫡

**Reply by aliok** [reply]
Thanks @halil-bugol !

Let's find a few more folks (at least 1 more, but I think everything would be easier if we're 5-6 people)

**Comment by boramir** [comment]
I would like to volunteer to both learn something new and contribute to the community. 🫡

**Comment by developer-guy** [comment]
count me in as you wish I'd love to help 🙋‍♂️

**Comment by mertsaygi** [comment]
I'm also in, how do we start?

**Comment by developer-guy** [comment]
I'm assuming @dentrax also would be happy to collaborate with us on this topic!

**Comment by aliok** [comment]
Let's wait until Wednesday for more people and then we can create a "team init" issue like this one: https://github.com/cncf/glossary/issues/305

**Comment by eminalemdar** [comment]
You can also count me in @aliok

**Comment by onurkcbyk** [comment]
I can volunteer to learn something new

**Comment by aliok** [comment]
Just created https://github.com/cncf/glossary/issues/2123
